storage/scylla_api.c

```diff
@@ -453,7 +454,11 @@ static status_t create_identity_table(CassSession* session, bool need_drop) {
     ret = SC_STORAGE_CASSANDRA_QUREY_FAIL;
     goto exit;
   }
-
+  if (execute_query(session, "CREATE INDEX IF NOT EXISTS ON identity(hash);") != CASS_OK) {
+    ta_log_error("create identity table index fail\n");
+    ret = SC_STORAGE_CASSANDRA_QUREY_FAIL;
+    goto exit;
```

Comment:
We can simply return `ret` directly in this function.

Reply:
OK.